src/Admin/BulkEdit/CouponBulkEdit.php

```diff
@@ -138,6 +137,8 @@ public function handle_submission( int $post_id, $post ): int {
 	 * @return array The $_REQUEST superglobal.
 	 */
 	protected function request_data(): array {
+		// Nonce must be verified manually.
+		//phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
❓ 
This chunk of code seems to be unrelated with the `is_ajax()`. Do I need to test anything here?

Reply:
Nope, but it was giving a warning when I ran PHPCS so I added it to have a clean slate!